src/components/table/JadeTable.jsx

```diff
@@ -77,6 +77,7 @@ export class JadeTable extends React.PureComponent {
     classes: PropTypes.object.isRequired,
     columns: PropTypes.arrayOf(PropTypes.object),
     handleEnumeration: PropTypes.func,
+    items: PropTypes.string.isRequired,
```

Comment:
this isn't an array? Or if this is just a label for the type of items being shown maybe call it something else like itemType?